agent/util-scripts/gold/test-start-stop-tool-meister/test-51.txt

```diff
@@ -16,23 +16,12 @@ pbench-tool-meister-start: 5. starting tool data sink
 pbench-tool-meister-start: 6a. starting localhost tool meister
 pbench-tool-meister-start: 7. waiting for all successfully created Tool Meister processes to show up as subscribers
 pbench-tool-meister-start: 8. Initialize persistent tools
-pbench-tool-meister-start: channel pbench-agent-cli-to-client payload, '{"action": "init", "kind": "ds", "status": "success"}'
 pbench-tool-meister-start: channel pbench-agent-cli-to-client payload, '{"action": "startup", "kind": "ds", "status": "success"}'
 pbench-tool-meister-start: next pbench-agent-cli-to-client
-pbench-tool-meister-start: next pbench-agent-cli-to-client
```

Comment:
This is expected. "Startup" is still handled in pbench-tool-meister-start using the traditional method. The new client, however, has been switched over to state-signals, so the "init" signal is the first action now handled by state-signals rather than the traditional method. This is why we see the traditional debug logging for the action "startup" still, but no longer for "init".